src/app/chart/bar/bar.component.ts

```diff
@@ -122,6 +122,50 @@ export class BarComponent implements AfterViewInit {
     this.settings.ctrlsOpen = !this.settings.ctrlsOpen;
   }
 
+  /** roundUpNumber
+  /* Rounds up the number
+  /*
+  /* @param { number } num - the number to round
+  /* @returns number
+  */
+  roundUpNumber(num: number): number {
+    return Math.ceil(num / 10) * 10;
+  }
+
+  /** addAxisBreak
+  /* Adds an axis break if necessary
+  /*
+  /* @param { number } seriesMin - the smallest value in the series
+  /* @param { number } seriesMax - the largest value in the series
+  */
+  addAxisBreak(seriesMin: number, seriesMax: number): void {
```

Comment:
Instead of having a method void that does the check it self, I would prefer for this method to be dedicated to **only** do the axis break, and the methods who use it do the check themselves if it is needed to apply the break. 
Otherwise there will be moments where this method is called and the break is not even necessary.

Reply:
Changed